class CustomError(Exception):
 pass

raise CustomError("Example of Custom Exception")
